src/bo4e/bo/buendelvertrag.py

```diff
@@ -27,5 +34,35 @@ class Buendelvertrag(Geschaeftsobjekt):
 
     # required attributes
     bo_typ: BoTyp = BoTyp.BUENDELVERTRAG
+
+    # pylint: disable=duplicate-code
+    #: Eine im Verwendungskontext eindeutige Nummer für den Vertrag
+    vertragsnummer: str
+    #: Hier ist festgelegt, um welche Art von Vertrag es sich handelt. Z.B. Netznutzungvertrag
+    vertragsart: Vertragsart
+    #: Gibt den Status des Vertrages an
+    vertragsstatus: Vertragsstatus
```

Comment:
hier hast du nicht ganz umgesetzt, was in der doku war ("vertragstatus" mit nur einem s). aber ich finde doppel-s besser.

Reply:
Oh, Tatsache. Wie seltsam aber, dass das ENUM aber dann `Vertragsstatus` heißt.
Ich würde davon ausgehen, dass das 'n Typo ist, schließlich ist der Rest (`vertragsart`, `vertragsbeginn`, etc.) auch korrekt mit nem `s` dekliniert ;P

Soll ich das dann jetzt entsprechend der Doku mit einem `s` machen?